src/views/settings/SettingsSubjects.tsx

```diff
@@ -315,39 +317,107 @@ const SettingsSubjects: Screen<"SettingsSubjects"> = ({ navigation }) => {
                       keyExtractor={(item) => item}
                       ListFooterComponent={<View style={{ width: 16 }} />}
                       showsHorizontalScrollIndicator={false}
-                      renderItem={({ item }) => (
-                        <TouchableOpacity
-                          onPress={() => handleSubjectColorChange(selectedSubject[0], item)}
-                        >
-                          <View
-                            style={{
-                              width: 32,
-                              height: 32,
-                              borderRadius: 80,
-                              backgroundColor: item,
-                              marginHorizontal: 5,
-                              alignItems: "center",
-                              justifyContent: "center",
-                            }}
+                      renderItem={({ item }) => {
+                        return (
+                          <TouchableOpacity
+                            onPress={() => handleSubjectColorChange(selectedSubject[0], item)}
                           >
-                            {selectedSubject[1].color === item && (
-                              <Reanimated.View
+                            <View
+                              style={{
+                                width: 32,
+                                height: 32,
+                                borderRadius: 80,
+                                backgroundColor: item,
+                                marginHorizontal: 5,
+                                alignItems: "center",
+                                justifyContent: "center",
+                              }}
+                            >
+                              {selectedSubject[1].color === item && (
+                                <Reanimated.View
+                                  style={{
+                                    width: 26,
+                                    height: 26,
+                                    borderRadius: 80,
+                                    backgroundColor: item,
+                                    borderColor: colors.background,
+                                    borderWidth: 3,
+                                  }}
+                                  entering={ZoomIn.springify().mass(1).damping(20).stiffness(300)}
+                                  exiting={ZoomOut.springify().mass(1).damping(20).stiffness(300)}
+                                />
+                              )}
+                            </View>
+                          </TouchableOpacity>
+                        );
+                      }}
+                    />
+                    {(
+                      <MemoizedNativeList>
+                        <MemoizedNativeItem>
+                          <MemoizedNativeText variant="subtitle" numberOfLines={1}>
+                            Écrire la couleur (6 caractères)
```

Comment:
Pour éviter que l'utilisateur fasse ça en premier :

```suggestion
                            Ou écrire la couleur (6 caractères)
```